bin/docker_start

```
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'uri'
+
+# check that postgres is up
+pg_uri = URI.parse(ENV['DATABASE_URL'])
+`dockerize -wait tcp://#{pg_uri.host}:#{pg_uri.port || 5432} -timeout 60s`
```

This script will silently ignore errors

---

added error handling